model/clusters_mgmt/v1/log_forwarder_application_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Represents an application that can be configured for log forwarding.
+struct LogForwarderApplication {
+    // The name of the application.
+    Name String
```

Comment:
@willkutler I would make this `id`, so that it _could_ be referenced in a path at somepoint e.g. `/log_fowarding/applications/{id}`. See how we do this for `/api/cluster_mgmt/v1/versions`

`name` I think would be better reserved for a human readable description in the future alongside `description`, which is not in scope at the minute.